depot/containerstore/volume_mounted_files_handler.go

```diff
@@ -0,0 +1,172 @@
+package containerstore
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"code.cloudfoundry.org/executor"
+	"code.cloudfoundry.org/garden"
+	"code.cloudfoundry.org/lager/v3"
+)
+
+//go:generate counterfeiter -o containerstorefakes/fake_volume_mounted_files_handler.go . VolumeMountedFilesImplementor
+type VolumeMountedFilesImplementor interface {
+	CreateDir(logger lager.Logger, container executor.Container) ([]garden.BindMount, error)
+	RemoveDir(logger lager.Logger, container executor.Container) error
+}
+
+type VolumeMountedFilesHandler struct {
+	fsOperations       FSOperator
+	containerMountPath string
+	volumeMountPath    string
+}
+
+func NewVolumeMountedFilesHandler(
+	fsOperations FSOperator,
+	volumeMountPath string,
+	containerMountPath string,
+) *VolumeMountedFilesHandler {
+	return &VolumeMountedFilesHandler{
+		fsOperations:       fsOperations,
+		volumeMountPath:    volumeMountPath,
+		containerMountPath: containerMountPath,
+	}
+}
+
+func (h *VolumeMountedFilesHandler) CreateDir(logger lager.Logger, container executor.Container) ([]garden.BindMount, error) {
+	err := h.fsOperations.Chdir(h.volumeMountPath)
+	if err != nil {
+		return nil, fmt.Errorf("volume mount path doesn't exists %w", err)
```

Comment:
Ugh, sorry, found one last typo.

```suggestion
		return nil, fmt.Errorf("volume mount path does not exist %w", err)
```

❓ Can you update the tests to match this new error message?

Reply:
@ameowlia Done, feedback applied.